src/forms.rs

```diff
@@ -1,6 +1,93 @@
 use super::*;
 
-/// A form field.
+/// Writer for an _interactive forms dictionary_. PDF 1.2+.
+///
+/// This struct is created by [`Catalog::form`].
+pub struct Form<'a> {
+    dict: Dict<'a>,
+}
+
+writer!(Form: |obj| Self { dict: obj.dict() });
+
+impl<'a> Form<'a> {
+    /// Write the `/Fields` attribute to reference the root [form fields](Field)
+    /// (those who have no immediate parent) of this document.
+    pub fn fields(&mut self, fields: impl IntoIterator<Item = Ref>) -> &mut Self {
+        self.dict.insert(Name(b"Fields")).array().items(fields);
+        self
+    }
+
+    // TODO: deprecated in PDF 2.0
+
+    /// Write the `/NeedAppearances` attribute to set whether to construct
+    /// appearance streams and appearance dictionaries for all widget
+    /// annotations in this document.
+    pub fn need_appearances(&mut self, need: bool) -> &mut Self {
+        self.dict.pair(Name(b"NeedAppearances"), need);
+        self
+    }
```

Comment:
It was noted that this was deprecated in 2.0, I can't quite remember if it should've been included anyway.